snap/snapenv/snapenv.go

```diff
@@ -127,17 +125,18 @@ func basicEnv(info *snap.Info) map[string]string {
 // used by so many other modules, we run into circular dependencies if it's
 // somewhere more reasonable like the snappy module.
 func userEnv(info *snap.Info, home string) map[string]string {
-	// TODO parallel-install: use of proper instance/store name
+	// TODO parallel-install: we do not have a way to make the mounts from
+	// instance-specific to snap-specific directories at user controlled
+	// location completely safe, make sure we use instance-specific
+	// directories always
```

Comment:
Given what @zyga described, I'm not sure this is true. I'd simplify this comment to just saying:
```
// To keep things simple the user variables always point to the
// instance-specific directories.
```